NAMESPACE

```diff
@@ -37,21 +37,30 @@ export(simulate_exponential_cases)
 export(simulate_gillespie)
 export(simulate_secondary)
 export(simulate_uniform_cases)
-import(brms)
```

Comment:
if we have specify imports from brms then remove the global import of brms

Reply:
This is in `epidist-package.R` right?

Reply:
yes

Reply:
Removed in [f509890](https://github.com/epinowcast/epidist/pull/329/commits/f509890151e0779bc053a967ed3de8edaa73bae1)